vividus/src/main/java/org/vividus/transformer/AbstractFilteringTableTransformer.java

```diff
@@ -31,9 +33,16 @@
     protected List<String> filterColumnNames(List<String> allColumnNames, String byColumnNames)
     {
         List<String> filtered = new ArrayList<>(allColumnNames);
```

Comment:
it makes sense to move it after the validation to avoid its creation in case validation failure